.github/workflows/release.yml

```diff
@@ -166,6 +166,39 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
         if: ${{ !contains(github.event.inputs.releasing, '-') }}
 
+  unpublish-prebuilt:
+    name: Unpublish prebuilt platform-specific packages
+    runs-on: ubuntu-latest
+    needs: [build-publish-prebuilt, x64-group-install]
+    if: ${{ always() && contains(needs.*.result, 'failure') }}
+    strategy:
+      matrix:
+        node: ['14', '16', '18']
+
+    steps:
+      - name: Setup Node ${{ matrix.node }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          registry-url: 'https://registry.npmjs.org' # Setup .npmrc file to unpublish from npm
+
+      - name: Unpublish arm64-linux-gnu-${{ matrix.node }}
+        run: npm unpublish arm64-linux-gnu-${{ matrix.node }}@${{ github.event.inputs.releasing }}
```

Comment:
Should it be `.. unpublish solarwinds-apm-bindings-arm64-linux-gnu...` instead?

Reply:
good catch